### DISCUSSION TITLE
Not send E-mails end of file reached

### DISCUSSION THREAD

**Discussion by javierllovet** [postalserver/postal#2198] [Help with using Postal]
When we try to send e-mails still generate Soft Fails and still got the error End of File Reached

We check and we are not in Blacklist or other blocking issues

This only happen with Microsoft, and we read in th ohtr fourms this could be something about changes in their service or something on the libraries use on Rails.

We have 5 days in a row and we cannot get out to them, if there something else we can do or review or modify

![image](https://user-images.githubusercontent.com/73400666/197359222-4d6c63ef-c38a-4352-83fb-c31f1ac72124.png)

![image](https://user-images.githubusercontent.com/73400666/197359227-23f4bad3-72cd-4b39-a37e-8502e59111b6.png)

**Comment by willpower232** [comment]
as I mentioned in #2192, you need to ask your server provider company if there are any blocks in place with them

If Postal says there were no SMTP servers available then it could not connect, either microsoft is blocking your server provider or your server provider has some incomplete block in place. There is not a problem with Postal here.
